Godeps/Godeps.json

```diff
@@ -1115,317 +1115,322 @@
 		},
 		{
 			"ImportPath": "github.com/docker/distribution",
-			"Comment": "v2.6.2",
-			"Rev": "48294d928ced5dd9b378f7fd7c6f5da3ff3f2c89"
+			"Comment": "v2.6.2-7-g729840c",
+			"Rev": "1e2bbed6e09c6c8047f52af965a6e301f346d04e"
```

Comment:
a bump commit consisting only of a sha change doesn't seem correct... there should be file changes associated with this, right?

Reply:
These are bumps that match the synced repos in our forks, that David didn't push to origin.